weave/trace_server/migrations/002_add_deleted_at.down.sql

```diff
@@ -4,50 +4,6 @@ This migration undoes adding the `deleted_at` column to:
     - the object_versions_deduped and calls_merged_view views
 */
 
-ALTER TABLE object_versions DROP COLUMN deleted_at;
```

Comment:
IGNORE: will pull out of PR, this is just what I had to do to get my dev setup working